bin/smalltalkci

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env bash
 
-if [[ -z "${SMALLTALK_CI_HOME:-}" || "$@" = *--self-test* ]]; then
+if [[ -z "${SMALLTALK_CI_HOME:-}" || "$*" = *--self-test* ]]; then
```

Comment:
Does this fix an actual problem?

Reply:
It was reported by shellcheck:  `SC2199: Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).`